website/backend/settings.py

```diff
@@ -141,11 +142,16 @@
 # From Django's perspective, this is the input location.
 STATICFILES_DIRS = []
 
-if DEBUG:
-    STATIC_HOST = 'http://localhost:8081'  # Use webpack-dev-server on port 8081
-    STATICFILES_DIR = []  # type: Sequence[str]
+if not DEBUG:
+    GS_CREDENTIALS = service_account.Credentials.from_service_account_file(
+        os.path.join(BASE_DIR, 'google_application_credentials.json')
+    )
 
-STATIC_URL = STATIC_HOST + '/static/'
+    DEFAULT_FILE_STORAGE = 'storages.backends.gcloud.GoogleCloudStorage'
+    GS_BUCKET_NAME = 'airqo-website'
```

Comment:
We might need to pick this from the `.env`

Reply:
This has been updated